contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -299,6 +310,27 @@ NXmpes(NXobject):
           exists: recommended
           doc: |
             This is the momentum axis to be used for data plotting.
+      energy_correction(NXcalibration):
```

Comment:
This is probably still too long, but you need all of these fields for such kind of energy correction. The rationale for having this here is that often (at least in XPS), the energy axis is adjusted by shifting all energies uniformally until one peak (or the Fermi edge) is located at a known "correct" energy.